app/Entry.js

```diff
@@ -28,58 +27,59 @@ import { SettingsScreen } from './views/Settings';
 
 const Stack = createStackNavigator();
 
-export const Entry = () => {
-  const [onboardingDone, setOnboardingDone] = useState(false);
+const screenOptions = {
+  cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
+  cardStyle: {
+    backgroundColor: 'transparent', // prevent white flash on Android
+  },
+  headerShown: false,
+};
 
-  useEffect(() => {
-    async function checkDone() {
-      /** @type {boolean | undefined | null} */
-      const flag = await GetStoreData(ONBOARDING_DONE, false /* isString */);
-      setOnboardingDone(!!flag);
-    }
+const MainApp = () => (
```

Comment:
This abstraction will make swapping out for a Tab Navigator easier in upcoming design changes